README.md

```diff
@@ -228,6 +228,20 @@ Read in a different language: [![CN](/assets/flags/CN.png)**CN**](/README.chines
 
 <p align="right"><a href="#table-of-contents">⬆ Return to top</a></p>
 
+## ![✔] 2.12 Always await promises before returning
+
+**TL;DR:** Always do `return await` when returning a promise. If a function returns a promise, that  
+function must be declared as async function and explicitly await the promise before returning it
+
+**Otherwise:** The function would be missed in stacktraces and errors would look like they came out of  
+something like `process.nextTick` in to a random place in code
```

Comment:
What about rephrasing with something like `from a random place in node internal code` ?

Reply:
I don't want to have two "random" one after another, and, actually nothing there is random :) so I have rephrased it a little bit differently. Hope new wording sounds better

Reply:
This otherwise sentence is technically true but not simple enough. Maybe:

'The function would not be part of the StackTrace and it will be harder to follow an error path without some of the calls chain'